flexmeasures/data/models/validation_utils.py

```diff
@@ -59,3 +68,38 @@ def check_required_attributes(
         raise WrongTypeAttributeException(
             f"Sensor attributes are not of the required type:\n {error_message}"
         )
+
+
+def can_modify_role(current_user: "User", roles_to_modify: list["Role"]):  # noqa: F821
```

Comment:
Here, the first argument should just be called `user`, we can use this for anyone. Also adjust the docstring.